src/tests/ingestion.int.test.js

```diff
@@ -0,0 +1,45 @@
+// Mocked import
+import childProcess from 'child_process' // eslint-disable-line import/order
+
+// Test imports
+import path from 'path'
+import { PassThrough } from 'stream'
+import FileIngestionEngine from '%src/components/ingestion/FileIngestionEngine'
+
+jest.mock('child_process')
+
+describe('ingestion', () => {
+	describe('FileIngestionEngine', () => {
+		it('it should read a video file into a Kafka topic', (done) => {
+			jest.clearAllMocks()
+
+			const mockDate = new Date(1592156234000)
+			jest
+				.spyOn(global, 'Date')
+				.mockImplementation(() => mockDate)
```

Comment:
oh snap this is great ty for showing this approach

Reply:
Thank you! I got it from StackOverflow.